index.js

```diff
@@ -2,7 +2,11 @@
 
 const app = require('./dist/app');
 
-app.start(process.argv.slice(2)).catch(error => {
+const args = process.argv
+  .slice(2)
+  .reduce((acc, val) => acc.concat(val.split('=')), []); // support both "key=val" and "key val"
```

Comment:
We don't need to communicate the usage of this logic in an inline-comment

Reply:
I'm also not 100% convinced we need to pre-process the flags right now. Can you explain the need?

Reply:
it is not necessary, only if you want to provide `--count=2`  this prepossessing is needed.
`--count 2` will work just fine even without the preprocessing.

Reply:
I often use both the formats while providing args to CLI tools and most tools support it.
I'll remove it , if you think it is not necessary

Reply:
Adding to the conversation instead of the new commits: I don't think it is necessary in this case with as few arguments as we have. The way we currently have our options for CIO set up, its a very simple statement and will not work if we add more options in the future.

My reasoning for not wanting to add this line in is that when we get around to adding in a second option, this will break and we won't be able to parse options. I had plans to introduce a cli parsing library at that point. Though it would require some organization of this file.

Reply:
modified.